Gruntfile.js

```diff
@@ -108,11 +136,12 @@ module.exports = function(grunt) {
     templateConfig[demo.slug] = {
       options: {
         data: {
-          header_begin: getTemplate('JavaScript/common/header_begin.html'),
-          header_end: getTemplate('JavaScript/common/header_end.html'),
-          footer: getTemplate('JavaScript/common/footer.html'),
-          ga_script: getTemplate('JavaScript/common/ga_script.html'),
-          segment_script: getTemplate('JavaScript/common/segment_script.html')
+          demo_root: demo.slug,
```

Comment:
:bulb: An idea–could we call this something like `asset_prefix` versus `demo_root`? Took me a bit to figure out what role that variable was playing before I realized it ensured static assets were resolve-able without the trailing slash.

Reply:
Yep, I'll update. I was on the fence with that name anyway. 👍